apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -0,0 +1,66 @@
+import {postOpenaiChatCompletion} from '@cdo/apps/aichat/chatApi';
+import {Role} from '@cdo/apps/aichat/types';
+import {createSlice, PayloadAction, createAsyncThunk} from '@reduxjs/toolkit';
+
+const registerReducers = require('@cdo/apps/redux').registerReducers;
+
+export interface AiTutorState {
```

Comment:
Can you clarify why this is `AiTutorState` and the one below is `AITutorPanel`?

I _think_ the folks working on EduBot landed on `Aichat` everywhere, presumably for consistency, given how Rails capitalization works. @molly-moen and I had a similar conversation about Javalab vs. Java Lab that I now can't find documentation for. Would `Aitutor` everywhere be reasonable?

Reply:
Hi Emily, since you linked me to this comment -- I would argue that `AITutorPanel` should at least be changed to `AiTutorPanel` (lower case `i`) -- then I think it is up to your team whether to go further and pursue the `Aitutor` approach.

Reply:
Thanks, and that makes sense! I'm on board with everything that gets us more consistent naming 😄

Reply:
I don't really know that it's worth it to go down this rabbit hole, but the more I think about it the more I care about consistency between the A and the I ("AI" or "ai" rather than "Ai"). My thought is that AI is an abbreviation of a proper noun so it should, in everyday English, be capitalized and conceptually the two letters go together in reference to one thing so their casing should be the same. 🤷‍♀️

Reply:
@Erin007 What do you think about `Aitutor`, consistent with `Ailab` and `Aichat`, and how many of the cases in this PR does it solve?

I went down the rabbit hole for a couple of minutes and tried to check what's in our style guide: https://github.com/code-dot-org/code-dot-org/blob/6633d79db43fbc66d9349aff1220a188c8b7b42c/STYLEGUIDE.md#es6. We have no explicit notes about capitalization. We note that we follow Airbnb's style guide as a fallback, and their capitalization rules agree with @Erin007: https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms.

Personally, consistency across the code base is the most important thing to me.

Reply:
^ I just made that up based on personal preference but fwiw the airbnb styleguide agrees: https://github.com/airbnb/javascript#naming--Acronyms-and-Initialisms

Reply:
Yeah, I noted the agreement with you in my comment above :) `Aitutor` is not something we agree on for capitalization rules; it's a decision we make about (it looks like) game types to keep capitalization consistent between the front-end and the back-end: https://github.com/code-dot-org/code-dot-org/pull/52538/files#diff-fb10c591e5aa0c82109ab9b5e3d5aac6f7ffdf5ebb2717de405afa7c661dd650R445. Since it looks like it's tied to game type, I don't think it applies here, and we're back to `AITutor` vs. `aiTutor`. Given this isn't a new game type, I have no strong feeling as long as naming conventions within the PR are consistent.